contents/blog/decouple-deployment-from-release.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Why (and how) you should decouple deployment from release
+date: 2023-09-07
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+You build software for others to use. This means you need a process for getting that software to those people. This is easier than ever thanks to the internet, hosting services, code management platforms, integration pipelines, and more innovations.
+
+What remains a challenge is delivering code to people **safely**. A best practice for doing this is decoupling deployment and release, but what does this even mean? This post goes over the differences between the two, why you should decouple them, examples, and how to get started.
+
+## What are the differences between deployment and release?
+
+The process of delivering code starts with writing and testing code locally. After completing a piece of code, it often goes through reviews and more tests before integrating with existing code. 
+
+Once integrated, the new code deploys. Deployment means getting code from a local machine or repository into production. Your application or build containing the new code becomes accessible on production infrastructure.
+
+After deploying successfully, the code releases for users to start using it. The application exposes in production and the functionality becomes available. A release comes after a deployment, and by default, couples to it. This means it happens immediately after a deployment succeeds.
+
+![Delivery process](../images/blog/decouple-deployment-from-release/delivery.png)
+
+This whole process is the software delivery process. Deployment and release are two separate, but critical parts of it.
```

Comment:
Do we really need to say this is the software delivery process. Feels a bit dull

```suggestion
```

Reply:
Removed